### ISSUE TITLE
Add a "link to discussion forum" to direct contribution

### ISSUE THREAD

**Issue by davidascher** [darrinhenein/cupcake-dashboard#26]
Hive uses discourse as a place to have discussion.  It'd be good to have an easy way to link projects to those threads.  Easiest today is just to have a link in the description, but probably worth generalizing the bugzilla link to be a not-so-bugzilla-specific notion of "look over _here_ to see & join the discussion". /cc @iamjessklein @toolness @keyboardkat

**Comment by iamjessklein** [comment]
:+1:

**Comment by toolness** [comment]
This would be cool--though just FYI different hives use different channels for discussion. NYC, Toronto, and Chicago all use [minigroup](http://minigroup.com/), though I'd love it if they migrated to Discourse.

**Comment by iamjessklein** [comment]
I wonder if it could be as simple as allowing for a link to where discussion is happening. So it could be the discourse link, a blogpost, a twitter hashtag.

**Comment by toolness** [comment]
Yeah, I think that is what david was ultimately suggesting, actually. I was just clarifying that not all Hives use Discourse. :)

**Comment by keyboardkat** [comment]
Thanks for filing this David! Would be a great addition to make Cupcakes more public/community-friendly.
